### ISSUE TITLE
Fedora template is not compatible with epel

### ISSUE THREAD

**Issue by TomasTomecek** [fedora-python/pyp2rpm#157]
```
error: line 53: Unknown tag: %python_provide: ERROR: python3-conu not recognized.
```

Would be awesome to have a single template which can be used for both Fedora and epel -- I don't want to maintain two specfiles.

**Comment by hroncok** [comment]
EPEL and Fedora are different. Trying to create one general specfile usually creates an unnecessary maintenance problem when we try to enforce new python packaging rules in Fedora. Hence, the Fedora template is deliberately not compatible with EPEL (I mean, we are not doing crazy things just to break EPEL, we just don't look at EPEL when making decisions about the template).

Is the EPEL7 template good enough for you usecase? Can it be improved to be?

**Comment by TomasTomecek** [comment]
This is what I did: https://github.com/fedora-modularity/conu/pull/96/files

tl;dr: disabled python3 build, wrapped that `%python_provide` in `%if 0%{?fedora}` and finally had to pick some other theme for sphinx since rtd wasn't available.

Other than that, the epel7 template was indeed helpful. It's just that I want to maintain a single specfile.

**Comment by hroncok** [comment]
BTW the `%{?python_provide:...` should already be conditionalized by the existence of the macro itself, I don't know why RPM fails here.

**Comment by hroncok** [comment]
Closing as duplicite https://github.com/fedora-python/pyp2rpm/issues/68
